### DISCUSSION TITLE
crash my game

### DISCUSSION THREAD

**Discussion by 2020oreo** [maruohon/litematica#665] [Q&A]
![crash](https://user-images.githubusercontent.com/124100009/216171441-3b612eab-8465-45d6-86e5-42adc0b24bba.png)
hello when i tried to load a shematica in my 1.18.2 world with fabric and the mods version is 0.11.3 my game crash and every time i do dats it crash can someone help me?

**Comment by maruohon** [comment]
Nobody knows why it crashes unless you post the crash log. (post it to some pastebin and link it here.)

But it's possible this is the recent GPU driver related crash. In that case you should update to version `0.11.5`: https://www.curseforge.com/minecraft/mc-mods/litematica/files/4259736

**Reply by 2020oreo** [reply]
I just post the screen.

**Reply by maruohon** [reply]
That short error message is not enough. I need to see the actual crash log. You should have a file at `.minecraft/crash-reports/` with a corresponding date and time to when the game crashes, or if there is nothing there, then check the `.minecraft/logs/latest.log` file if it has the crash. Post that log to some pastebin service and link it here.

**Reply by 2020oreo** [reply]
here is the pastbin
https://pastebin.com/tFZifCzh

**Reply by maruohon** [reply]
When exactly does it crash? Is it immediately when you try to open the `Load Schematics` menu, or later when you change directories in there or something?

**Reply by 2020oreo** [reply]
It crash when i click load shematics. I see the word quiting animation (saving world) then i crash. Here is a video of the crash.
https://www.mediafire.com/file/gbypmlrz6h0amfe/20230203_170022.mp4/file

**Reply by maruohon** [reply]
You may have an invalid directory saved in the config file. Have you deleted some schematic directories or moved the configs or Minecraft directories around?

But basically to fix this try editing or removing the config file from `.minecraft/config/litematica/<yourworldorservername>*.json`.

**Reply by 2020oreo** [reply]
Thanks for you helps it work 👍

**Comment by alexandr-create** [comment]
I can already confidently call myself a gamer, because I have tried many games. Now I can make a rating of the most interesting ones. Recently I found [worldle game]([https://worldlegame.io/](https://worldlegame.io/?)), I really liked it. The developers have invested a lot of effort to make it accessible to all ages. I recommend trying to spend an evening with this game.

**Comment by pimerdederde** [comment]
here is my paste bin

https://pastebin.pl/view/7cc321ed

**Reply by sakura-ryoko** [reply]
> here is my paste bin
> 
> https://pastebin.pl/view/7cc321ed

You appear to be using -sakura.5 before the version of litematica was updated to match.  Use MaLiLib -sakura.3 or below for 1.21.4.

Also, this thread has nothing to do with this.
